doc/SensorFusion/SRS/srs_eng.md

```diff
@@ -0,0 +1,385 @@
+# Software Requirements Specification (SRS)
+
+## Project Title: Sensorfusion of Localization data for Pololu Zumo32U4
+
+Author: Tobias Haeckel
+Bachelor Thesis
+
+## Table of Contents
+
+- [Software Requirements Specification (SRS)](#software-requirements-specification-srs)
+  - [Project Title: Sensorfusion of Localization data for Pololu Zumo32U4](#project-title-sensorfusion-of-localization-data-for-pololu-zumo32u4)
+  - [Version History](#version-history)
+  - [Table of Contents](#table-of-contents)
+- [1. Introduction](#1-introduction)
+  - [1.1 Purpose](#11-purpose)
+  - [1.2 Product Scope](#12-product-scope)
+  - [1.3 Definitions, Acronyms, and Abbreviations](#13-definitions-acronyms-and-abbreviations)
+  - [1.4 References](#14-references)
+  - [1.5 Overview](#15-overview)
+- [2. Overall Description](#2-overall-description)
+  - [2.1 Product Perspective](#21-product-perspective)
+  - [2.2 Product Features](#22-product-features)
+  - [2.3 Assumptions and Dependencies](#23-assumptions-and-dependencies)
+    - [2.3.1 Assumptions](#231-assumptions)
+    - [2.3.2 Dependencies](#232-dependencies)
+- [3. Requirements](#3-requirements)
+  - [3.1 External Interface Requirements](#31-external-interface-requirements)
+    - [3.1.1 User Interfaces](#311-user-interfaces)
+    - [3.1.2 Hardware Interfaces](#312-hardware-interfaces)
+    - [3.1.3 Software Interfaces](#313-software-interfaces)
+  - [3.2 Functional Requirements](#32-functional-requirements)
+    - [3.2.1 External Inputs \& Communication](#321-external-inputs--communication)
+    - [3.2.2 Sensor Fusion](#322-sensor-fusion)
+    - [3.2.3 Modes of Operation](#323-modes-of-operation)
+      - [3.2.3.1 Simulation Mode](#3231-simulation-mode)
+      - [3.2.3.2 Real-World Mode](#3232-real-world-mode)
+    - [3.2.4 Logging and Diagnostics](#324-logging-and-diagnostics)
+    - [3.2.5 Error Handling](#325-error-handling)
+      - [3.2.5.1 Functional Requirements](#3251-functional-requirements)
+      - [3.2.5.2 Status Levels](#3252-status-levels)
+      - [3.2.5.3 Status / Event Codes](#3253-status--event-codes)
+  - [3.3 Quality Attributes](#33-quality-attributes)
+    - [3.3.1 Performance](#331-performance)
+    - [3.3.2 Availability](#332-availability)
+    - [3.3.3 Standards \& Compliance](#333-standards--compliance)
+  - [3.4 Design- and Implementation Constraints](#34-design--and-implementation-constraints)
+    - [3.4.1 Installation](#341-installation)
+    - [3.4.2 Distribution](#342-distribution)
+- [4. Verification](#4-verification)
+  - [4.1 Verification Methods](#41-verification-methods)
+  - [4.2 Traceability (Requirements → Primary Method)](#42-traceability-requirements--primary-method)
+  - [4.3 Test Environment](#43-test-environment)
+- [5. Appendices](#5-appendices)
+  - [Appendix A: Glossary](#appendix-a-glossary)
+  - [Appendix B: Analysis Models](#appendix-b-analysis-models)
+  - [Appendix C: Open Issues (TBD)](#appendix-c-open-issues-tbd)
+
+# 1. Introduction
```

Comment:
There shall only be one top level heading

Reply:
Consider installing https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint